### PR TITLE
Remove `$` so one can copy the snippet directly

### PR DESCRIPTION
**Pull request by mobilutz**
With the `$` it is not possible to just copy the install code and paste it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,13 +13,13 @@ The `apps:free` Heroku CLI plugin shows the apps using free dynos or free data (
...
 
 ```
-$ heroku plugins:install @heroku-cli/plugin-free
+heroku plugins:install @heroku-cli/plugin-free
 ```
 
 With the plugin installed, you can run it without any arguments to fetch all apps across personal and any Heroku Team you’re a part of.
 
 ```
-$ heroku apps:free
```
Comment (saracope) [inline README.md]: This one should stay be the previous can be removed. Please update to add the $ here back.

**Review by saracope — CHANGES_REQUESTED** [review]
One small change requested

File: README.md (hunk 1 above)
Comment (mobilutz) [reply README.md]: Done

**Comment by mobilutz** [comment]
> One small change requested

@saracope Done

**Review by saracope — APPROVED** [review]

**Pull request merged**